sample/kotlin/build.gradle.kts

```diff
@@ -111,6 +111,15 @@ android {
         }
     }
     ndkVersion = Dependencies.Versions.Ndk
+
+    signingConfigs {
+        create("release") {
+            storeFile = File(project.rootDir, "sample-android.keystore")
+            storePassword = System.getenv("E2E_STORE_PASSWD")
+            keyAlias = "dd-sdk-android"
+            keyPassword = System.getenv("E2E_STORE_PASSWD")
```

Comment:
just checking that it is not a typo: key password is the same as store password?

Reply:
Yes, unfortunately android signing requires the same password for both :/